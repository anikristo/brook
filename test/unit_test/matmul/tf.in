4 2
0 0
1 1
2 3
1 0