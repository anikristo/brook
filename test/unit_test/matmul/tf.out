4 1
0.0
2.0
6.0
9.525e-07
